tests/test_cache/test_cache.py

```diff
@@ -50,6 +50,42 @@ def test_does_not_implement_missing(self):
         with self.assertRaises(NotImplementedError):
             cache.BaseCache(10).fetch('http://example.com/robots.txt')
 
+class TestBaseCacheLRUCacheWithMissingHandler(unittest.TestCase):
+    '''Tests LRUCacheWithMissingHandler'''
+
+    def calls_the_missing_function(self):
+        # missing ...
```

Comment:
Is this a leftover comment? I don't really think this needs a comment anyway.

Reply:
weird, no idea. removed.